code/game/machinery/transformer.dm

```diff
@@ -20,7 +20,7 @@
 	/// How long until the next mob can be processed
 	var/cooldown_timer
 	/// The created cyborg's cell chage
-	var/robot_cell_charge = 5000
+	var/robot_cell_charge = 5 MEGA JOULES
```

Comment:
```suggestion
	var/robot_cell_charge = 5 * STANDARD_CELL_CHARGE
```

This will avoid any future issues whenever someone decides to change that define.

An even better alternative would be to cause the borgs to spawn with an upgraded+ cell instead of just a modified upgraded one, which has 5 MJ capacity.

Reply:
The typepath of the upgraded+ cell is /obj/item/stock_parts/cell/upgraded/plus

Reply:
Looks like `new_borg.cell = new /obj/item/stock_parts/cell/upgraded/plus(new_borg, robot_cell_charge)` covers that, so it's most likely the var that's causing the drain